test/suites/common-tanssi-parachain/registrar/test_registrar_extrinsic_permissions.ts

```diff
@@ -0,0 +1,53 @@
+import "@tanssi/api-augment";
+
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+import type { KeyringPair } from "@moonwall/util";
+import type { ApiPromise } from "@polkadot/api";
+
+describeSuite({
+    id: "COMM0301",
+    title: "Registrar extrinsics permissions",
+    foundationMethods: "dev",
+    testCases: ({ it, context }) => {
+        let api: ApiPromise;
+        let alice: KeyringPair;
+        let bob: KeyringPair;
+        const paraId = 2001;
+
+        beforeAll(() => {
+            alice = context.keyring.alice;
+            bob = context.keyring.bob;
+            api = context.polkadotJs();
+        });
+
```

Comment:
Since the tests are (almost) identical we  can also use a single file: and doing something like:

` registerAlias = runtimeName.includes("light") ? polkadotJs.tx.containerRegistrar : polkadotJs.tx.registrar;
`

But I dont dislike this neither, I will leve it up to you

Reply:
I think it's much better than copy-paste, will change it.

Reply:
Fixed